packages/solidity-contracts/test/erc20Gateway.ts

```diff
@@ -30,7 +32,11 @@ type Fixture = Pick<
   | 'signers'
   | 'deployer'
   | 'initialTokenAmount'
-> & { fuelMessagePortalMock: MockFuelMessagePortal };
+> & {
+  fuelMessagePortalMock: MockFuelMessagePortal;
+  CRY: CRY;
```

Comment:
hey, we do have an integration test using 6 decimal tokens (usdt, usdc), here https://github.com/FuelLabs/fuel-bridge/blob/main/packages/integration-tests/tests/bridge_mainnet_tokens.ts

Reply:
Looks good but do you mean the test shouldn't be done here or that the token isn't required?
We need 6 decimals assets to be exported as listed assets by local fuel core to test on Fuel Explorer Bridge. Curren't ERC20 Deployer only deploys 18 decimal tokens (ETH and TKN), hence why a third was created.

Reply:
Yeah we don’t need the integration test we already have it

Reply:
@arthurgeron also we have a [script]( https://github.com/FuelLabs/fuel-bridge/blob/main/packages/solidity-contracts/deploy/hardhat/008.set_canonical_token_bytecode.ts) for using the 6 decimals tokens, that we utilize in the current integration test suite so once this script is ran, you'd essentially can utilize the usdc, usdt tokens using their [mainnet addresses](https://github.com/FuelLabs/fuel-bridge/blob/main/packages/solidity-contracts/protocol/constants.ts#L16)